<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eco-Fridge</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../나뭇잎.jpg</normaloff>../나뭇잎.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>235</width>
      <height>165</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Cafe24 Oneprettynight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>우유, 유제품</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>Cafe24 Oneprettynight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>음료, 주류</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Cafe24 Oneprettynight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>채소류</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Cafe24 Oneprettynight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>계란</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>Cafe24 Oneprettynight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>해산물</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>Cafe24 Oneprettynight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>육류</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>Cafe24 Oneprettynight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>과일, 견과류</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="font">
        <font>
         <family>Cafe24 Oneprettynight</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>기타</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cafe24 Oneprettynight</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
